workflow-templates/im-test-cypress-schedule.yml

```diff
@@ -0,0 +1,130 @@
+# Workflow Code: SurprisedHedgehog_v22    DO NOT REMOVE
```

Comment:
Make this a unique name and change the version to v1.

Reply:
updated the name and version.